dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -31,7 +31,7 @@ interface IAppUpgradeProps {
   fetchChartVersions: (id: string) => Promise<{}>;
   getApp: (releaseName: string, namespace: string) => Promise<void>;
   getBindings: () => Promise<IServiceBindingWithSecret[]>;
-  getChartVersion: (id: string, chartVersion: string) => Promise<void>;
+  getChartVersion: (id: string, chartVersion: string) => Promise<{}>;
```

Comment:
Why is it this change?

Reply:
we return the response now: https://github.com/kubeapps/kubeapps/pull/604/files#diff-32e6760e59b5e0433a9fae60151e0a8dR108